### PR TITLE
docs: add presto ssl config details

### PR DESCRIPTION
**Pull request by busgo**
### SUMMARY
Added prestosql data source ssl configuration instructions.

### ADDITIONAL INFORMATION

- [x] Has associated issue:
- [ ] Required feature flags:
- [ ] Changes UI
- [ ] Includes DB Migration (follow approval process in [SIP-59](https://github.com/apache/superset/issues/13351))
  - [ ] Migration is atomic, supports rollback & is backwards-compatible
  - [ ] Confirm DB migration upgrade and downgrade tested
  - [ ] Runtime estimates and downtime expectations provided
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API

### CODE REVIEW

**Comment by villebro** [comment]
@busgo please provide a full sentence under the "SUMMARY" title in the PR description of what this PR is doing.

File: docs/docs/databases/presto.mdx [hunk 1]
```diff
@@ -35,3 +35,16 @@ datasource. If you’re using an older version of Presto, you can configure it i
...
+``` 
+   {
+     "connect_args":
+    {"protocol": "https",
+     "requests_kwargs":{"verify":false}
+  } 
+}
+```
+
+
```
Comment (villebro) [inline docs/docs/databases/presto.mdx:50]: You need to remove this trailing whitespace:
image

**Review by villebro — APPROVED** [review]
LGTM after you fix the linting issue

File: docs/docs/databases/presto.mdx [hunk 2]
```diff
@@ -35,3 +35,14 @@ datasource. If you’re using an older version of Presto, you can configure it i
...
 ```
+
+SSL Secure extra add json config to extra connection information.
+
+``` 
+   {
+     "connect_args":
+    {"protocol": "https",
+     "requests_kwargs":{"verify":false}
+  }
```
Comment (rusackas) [inline docs/docs/databases/presto.mdx:46]:
```suggestion
  }
```

Comment (rusackas) [reply docs/docs/databases/presto.mdx:46]: removing trailing whitespace.

**Comment by rusackas** [comment]
Closing this one due to inactivity, and I've rebooted it here: https://github.com/apache/superset/pull/26986

**Comment by rusackas** [comment]
Thanks @busgo for taking this on!

**Pull request closed without merging**